This patch is Solaris-specific and thus not suitable for upstream.

--- Python-2.7.10/configure.ac.~2~	2015-05-10 10:17:06 -0800
+++ Python-2.7.10/configure.ac	2015-05-18 14:42:30 -0800
@@ -1106,7 +1106,7 @@
 	;;
 
     *)
-	OPT="-O"
+	OPT=""
 	;;
     esac
 fi
